pagerduty/event_orchestration_path_test.go

```diff
@@ -463,6 +542,9 @@ func TestEventOrchestrationPathGlobalUpdate(t *testing.T) {
 		},
 	}
 
+	// Before comparison, we sanitize the response to make it match the already sanitized request
+	sanitizeOrchestrationPath(resp.OrchestrationPath)
```

Comment:
1. I suggest you to please test `sanitizeOrchestrationPath` helper function as a unit, instead of introducing adaptation of the response in the middle of the test.

2. In this case the comparison mocked data should resemble the expected output data, instead of changing the output of `client.EventOrchestrationPaths.Update` to make it equal to the mocked `want` expected output, it's ok and the result is the same, however the order in this must happen is backwards. Thus, I ask you to please make the necessary updates @swbradshaw

Reply:
> In this case the comparison mocked data should resemble the expected output data
The `input` that goes into the `client.EventOrchestrationPaths.Update` is sanitized, and then used in the `want` variable.  Thus, the `want` variable is sanitized and the response is not.  The test will never pass anymore without this line. 

Ultimately the problem is this library is incomplete for Event Orchestration. In addition to sanitizing the request, it should also sanitized the response. Without this, there is extra code needed to fix the response (Example, the [TF code has all these flattenEventOrchestrationXXXXX functions](https://github.com/PagerDuty/terraform-provider-pagerduty/blob/master/pagerduty/event_orchestration_path_util.go) which should not be needed.)